Scripts/script-DeleteConext.yml

```diff
@@ -20,24 +20,39 @@ script: |
                       Type: entryTypes.error};
   }
   if (shouldDeleteAll) {
-      var keysToKeep = (args.keysToKeep) ? args.keysToKeep.split(',') : [];
+      var keysToKeep = (args.keysToKeep) ? args.keysToKeep.split(',').map(function(item) { return item.trim(); }) : [];
```

Comment:
You are using `.map`, then add `fromversion:3.5.0`

Reply:
why? map it is not es6..